### PR TITLE
Closes #423 total_irrad docstring

### LINKED ISSUE #423: irradiance.total_irrad signature is incorrect
Function signature states

` Determine diffuse irradiance from the sky on a tilted surface.`

However, the function returns a dataframe with all components of POA irradiance: beam, sky diffuse, ground reflected.

### PR DESCRIPTION
**Pull request by cwhanse**
pvlib python pull request guidelines
====================================

Thank you for your contribution to pvlib python!

You may submit a pull request with your code at any stage of completion, however, before the code can be merged the following items must be addressed:

 - [x] Closes #423
 - [x] Fully tested. Added and/or modified tests to ensure correct behavior for all reasonable inputs. Tests must pass on the TravisCI and Appveyor testing services.
 - [x] Code quality and style is sufficient. Passes ``git diff upstream/master -u -- "*.py" | flake8 --diff`` and/or landscape.io linting service.
 - [x] New code is fully documented. Includes sphinx/numpydoc compliant docstrings and comments in the code where necessary.
 - [x] Updates entries to `docs/sphinx/source/api.rst` for API changes.
 - [x] Adds description and name entries in the appropriate `docs/sphinx/source/whatsnew` file for all changes.

Please don't hesitate to ask for help if you're unsure of how to accomplish any of the above. You may delete all of these instructions except for the list above.

Brief description of the problem and proposed solution (if not already fully described in the issue linked to above):

### CODE REVIEW

**Comment by cwhanse** [comment]
Done I think.

File: pvlib/irradiance.py [hunk 1]
```diff
@@ -294,13 +294,20 @@ def total_irrad(surface_tilt, surface_azimuth,
                 model='isotropic',
                 model_perez='allsitescomposite1990', **kwargs):
     r"""
-    Determine diffuse irradiance from the sky on a tilted surface.
+    Determine total in-plane irradiance and it's beam, sky diffuse and ground
```
Comment (wholmgren) [inline pvlib/irradiance.py]: its, not it's.

**Comment by cwhanse** [comment]
I knew that :)

**Comment by wholmgren** [comment]
Yes, of course, I'm sure it was intended to test the reviewers -- thanks!

File: docs/sphinx/source/whatsnew/v0.5.2.rst [hunk 2]
```diff
@@ -16,6 +16,7 @@ Bug fixes
 * fixed redeclaration of test_simplified_solis_series_elevation (:issue:`387`)
 * physicaliam now returns a Series if called with a Series as an
   argument. (:issue:`397`)
+* corrected docstring for irradiance.total_irrad (:issue: '423')
```
Comment (mikofski) [inline docs/sphinx/source/whatsnew/v0.5.2.rst]: pls change single quotes around #423 to backticks

**Pull request merged**